src/services/ghost/GhostStreamingParser.ts

```diff
@@ -304,6 +304,21 @@ export class GhostStreamingParser {
 			const searchContent = match[1]
 			// Extract cursor position from replace content
 			const replaceContent = match[2]
+
+			// Validate that extracted content doesn't contain XML tags (indicates regex failure)
+			if (
+				searchContent.includes("<change>") ||
+				searchContent.includes("CDATA") ||
```

Comment:
I don't think this is correct, we dont want to forbid autocomplete in XML I think?